### ISSUE TITLE
Change console output text and style

### ISSUE THREAD

**Issue by EtagiBI** [KhronosGroup/glTF-Blender-IO#613]
Hello,

At the moment successful gltf/glb import gives a following console output:
```
CRITICAL Data are loaded, start creating Blender stuff
CRITICAL glTF import finished in X.Xs
```
In my view, these lines are quite confusing because the word "CRITICAL" usually indicates critical errors. It becomes even more misleading in Blender Python console because these lines appear there in red-pink color, which is - again - associated with errors. 

Is it possible to replace "CRITICAL" by "SUCCESS" and change the color to blue?

This feature request may seem a bit nit-picky, but when you have to debug a lot of code every single red line or "CRITITCAL" signal draws attention distracting from real problems.

**Comment by emackey** [comment]
What platform are you on?  And what version of Blender?

**Comment by julienduroure** [comment]
This should be solved when we will address #89 , but this is not planned yet (neither at short or medium term).
So I will probably patch importer logging way for avoid this

**Comment by EtagiBI** [comment]
> What platform are you on? And what version of Blender?

I'm on Windows 10 Pro with Blender 2.8 installed.
